ui/src/utils/categories.js

```diff
@@ -89,3 +94,24 @@ export const labels = {
     return msg`Unknown | Ghostery organizes tags by category. This is one of several tag categories.`;
   },
 };
+
+export const getCategoryKey = (category) => {
+  return colors[category] ? category : 'unknown';
+};
+
+export const getCategoryColor = (category) => {
+  return colors[getCategoryKey(category)];
+};
+
+export const getCategoryLabel = (category) => {
+  return labels[getCategoryKey(category)];
+};
+
+export const sortCategories = (_resolveCategoryName) => {
```

Comment:
You can avoid underscore prefix and just do:
```suggestion
export const sortCategories = (resolveCategoryName = ((a) => a)) => {
```